### ISSUE TITLE
Spacing of unnamed Widgets

### ISSUE THREAD

**Issue by wabu** [pyviz/panel#196]
If you create widgets without a name, there still will be an empty space allocated for the label, which is annoying in a row-based layout, as the widgets will not be aligned with internally labeled widgets like Buttons or Checkboxes:
```python
import panel as pn

row = pn.Row(pn.widgets.TextInput(placeholder='file name'),
             pn.widgets.Select(options=['a','b','c']),
             pn.widgets.Button(name='Action'),
             pn.widgets.Checkbox())
row
```
![unlabeld](https://user-images.githubusercontent.com/39164/50682243-86e12780-100e-11e9-9e37-f3c8e6091b6b.png)

It would be desirable to remove the label, if there is no name (or, if you prefer to make this behavior explicit, if name=None is passed).

- panel 0.3.1
- bokeh 1.0.0

**Comment by xavArtley** [comment]
Hi, I think this issue is more related to bokeh. 
https://github.com/bokeh/bokeh/issues/4817

**Comment by DancingQuanta** [comment]
And bokeh is busy with creating a layout PR that could pave a way to better implemenation of widgets.

**Comment by wabu** [comment]
Thanks. Closing as it seems to be an upstream issue.

**Comment by kcpevey** [comment]
<img width="990" alt="Screen Shot 2020-03-30 at 4 34 10 PM" src="https://user-images.githubusercontent.com/15659110/77964274-69981900-72a4-11ea-83d2-0bc3dbcbc4af.png">
This is indeed fixed with Bokeh 2.0.0 and Panel 0.9.3
